Upgrade notes for AWS Security Lake input parsing adjustments

danotorrey opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

Update the `UPGRADING.md` file to cover parsing changes made in https://github.com/Graylog2/graylog-plugin-enterprise/pull/6247.

/nocl Change log present in above-linked source PR.

kingzacko1 commented on UPGRADING.md:
```diff
@@ -28,6 +28,22 @@ to better align with the GIM schema and allow for targeted handling of AD SIDs.
+`vendor_event_description`: Contains the value which was previously present in the `message` log field.
+`event_created`: Contains the `time` log value.
+`event_source_input`: Contains the static value `aws_security_lake`.
+`vendor_version`: Contains the `metadata.product.versoin` log value.
```
```suggestion
`vendor_version`: Contains the `metadata.product.version` log value.
```

danotorrey commented:
Thanks a lot for the feedback and updates @kingzacko1 @ryan-carroll-graylog! This all looks good to me. 

I made a couple more small tweaks: Alphabetized the added fields, and also removed the `full_message` (that it's still there) description. I had added that thinking it would be helpful, but now with all the inputs in there, it seems to distract a bit and is possibly not that helpful. Let me know if y'all want to keep it in there, and we can add it back.

ryan-carroll-graylog commented on UPGRADING.md:
```diff
@@ -34,6 +34,63 @@ to better align with the GIM schema and allow for targeted handling of AD SIDs.
+- The message `timestamp` field is now set to the current Graylog system date/time, instead of the previously used log `time` value. The `event_created` field now contains the previous `time` value for backwards-compatibility.
+
+Added fields:
+`event_created`: Contains the `time` log value.
+`event_source_input`: Contains the static value `aws_security_lake`.
+`vendor_event_description`: Contains the value which was previously present in the `message` log field.
+`vendor_version`: Contains the `metadata.product.version` log value.
```
Nit: Noticed missing bullets here where we have them for the other sections @danotorrey 
```suggestion
- `event_created`: Contains the `time` log value.
- `event_source_input`: Contains the static value `aws_security_lake`.
- `vendor_event_description`: Contains the value which was previously present in the `message` log field.
- `vendor_version`: Contains the `metadata.product.version` log value.
```

danotorrey replied:
Thanks a lot for seeing this! Fixed.

danotorrey commented:
If this looks good in it's latest form to you @ryan-carroll-graylog @kingzacko1 then I will go ahead and merge this after the high five.
